pom.xml

```diff
@@ -122,7 +122,7 @@ under the License.
 		<zookeeper.version>3.4.14</zookeeper.version>
 		<!-- Only the curator2 TestingServer works with ZK 3.4 -->
 		<curator.version>2.12.0</curator.version>
-		<jackson.version>2.10.1</jackson.version>
+		<jackson.version>2.12.1</jackson.version>
```

Comment:
Is this always updated when the jackson version in flink-shaded is updated?

Reply:
yes, this effectively is the flink-shaded-jackson version.